src/index.css

```diff
@@ -16,21 +36,64 @@ nav ul {
 
 .booksul li {
   font-size: 20px;
-  padding-bottom: 1rem;
-  display: flex;
-  gap: 1rem;
+  gap: 0.938rem;
+  display: grid;
+  grid-template-columns: 2fr 1fr;
+  height: 10.625rem;
+  margin: 2.313rem 6.25rem 0;
+  padding: 2rem 4rem 7.625rem 1.688rem;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+  background-color: #fff;
 }
 
-.booksul li button {
-  padding: 3px 5px;
+nav ul li:nth-child(1) {
+  width: 3.688rem;
+  height: 1rem;
+  margin: 1.125rem 2.563rem 0.688rem 0;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 0.813rem;
+  letter-spacing: 1.9px;
+  color: #121212;
 }
 
-nav {
-  padding: 1rem 2rem;
-  background-color: rgb(219, 219, 219);
-  height: 50px;
-  display: flex;
-  gap: 4rem;
+nav ul li:nth-child(2) {
+  width: 6.563rem;
+  height: 1rem;
+  margin: 1.125rem 41.5rem 0.688rem 2.563rem;
+  opacity: 0.5;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 0.813rem;
+  letter-spacing: 1.9px;
+  color: #121212;
+}
+
+form button {
+  width: 11.5rem;
+  height: 2.813rem;
+  margin: 1.188rem 0 0 2.125rem;
+  border: none;
+  padding: 0.801rem 1.188rem 0.886rem 1.375rem;
+  border-radius: 3px;
```

Comment:
- [ ] Your effort on this task is commendable. However, you haven't properly replicated the form design as it should be. I would suggest you increase the width of the form `inputs` to align with the Zeplin design. 

- [ ] In addition, Kindly fix the `add book` button to align with the model design. This will not only improve the user interface of your app but will also fulfill the design requirements.

> Zeplin design
![reaL](https://user-images.githubusercontent.com/71140133/230498188-27942cdc-dcca-4817-bb77-e25a67f5d28a.PNG)

> Your design
![Bookform](https://user-images.githubusercontent.com/71140133/230498322-c68d87a1-616b-4f81-b37e-1233015b59fa.PNG)

Reply:
Helllo @Yerimah , thank you on all these insights. I have worked on them and surely learned a lot. 😊💯